1.prepare.sh

```diff
@@ -13,20 +13,26 @@ fi
 # Download klaytn docker image.
 docker pull klaytn/klaytn:latest
 
+CURR_PWD=`pwd`
+OUTPUT_DIR=/homi-output
+case "$(uname -sr)" in
+   CYGWIN*|MINGW*|MINGW32*|MSYS*)
+     CURR_PWD=`pwd -W`
+     OUTPUT_DIR=//homi-output
+     ;;
+esac
+
 # Generate docker-compose.yml for docker-compose.
-docker run --rm -v ${PWD}:/homi-output klaytn/klaytn homi setup -o /homi-output --docker-image-id klaytn/klaytn:latest --cn-num $NUM_CNS --chainID $CHAIN_ID --network-id $NETWORK_ID docker
+docker run --rm -v $CURR_PWD:/homi-output klaytn/klaytn homi setup -o $OUTPUT_DIR --docker-image-id klaytn/klaytn:latest --cn-num $NUM_CNS --chainID $CHAIN_ID --network-id $NETWORK_ID docker
 
 # Update private key if necessary
 if [ ! -z $PRIVATE_KEY ]; then
```

Comment:
Hi, Do we need this **PRIVATE_KEY block here** **[Update private key if necessary]**. If its required, i will revert back python file. i'm thinking to create a shell file instead of python file just to have consistency ? Please let me know your thoughts.

Reply:
I think your change is okay, but in this case, you also need to check $ADDRESS or generate address from the private key.

Reply:
sure @kjhman21 . Thanks. i will put back these details.

Reply:
@praveen-klaytn Let me clarify this.

1. PRIVATE_KEY can be updated whatever the user wants because this network can be also used with remix or other development tools.
2. `add_import_key.sh` is required to unlock the account so that later execution can work without any unlocking.


so, could you leave the `add_import_key.sh` execution as it is? outside of `if` statement?

Even if the user does not set `PRIVATE_KEY`, it could work with 'personal.unlockAccount`.

Reply:
As per the observation, without add_import_key.sh itself, by default **import and unlock commands** are added in docker-compose file with **autogenerated keys**. But If a PRIVATE_KEY is configured manually, only then the **add_import_key.sh** is required to add the commands to the docker-compose file.